common/metacity-1/button-border.svg

```diff
@@ -1,4 +1,5 @@
-﻿<svg viewBox="0 0 16 16">
+﻿<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16">
     <g id="titlebutton-max-hover" transform="translate(-513,185.63782)" inkscape:label="#g6284">

```

Comment:
This SVG is still invalid because of `inkscape:label="#g6284"`. You should test your changes before creating a pull request.